tests/codeception/functional/BasicPage/BasicPageParagraphsCest.php

```diff
@@ -10,19 +10,28 @@ class BasicPageParagraphsCest {
    */
   public function testCardParagraph(FunctionalTester $I) {
     $paragraph = $I->createEntity(['type' => 'stanford_card'], 'paragraph');
+
+    $row = $I->createEntity([
+      'type' => 'node_stanford_page_row',
+      'su_page_components' => [
+        'target_id' => $paragraph->id(),
+        'entity' => $paragraph,
+      ],
+    ], 'paragraph_row');
+
     $node = $I->createEntity([
       'type' => 'stanford_page',
       'title' => 'Test Cards',
       'su_page_components' => [
-        'target_id' => $paragraph->id(),
-        'entity' => $paragraph,
+        'target_id' => $row->id(),
+        'entity' => $row,
       ],
     ]);
     $I->logInWithRole('contributor');
     $I->amOnPage("/node/{$node->id()}/edit");
     $I->waitForElementVisible('#row-0');
     $I->click('Edit', '#row-0');
-    $I->waitForElement('.MuiAutocomplete-input');
+    $I->waitForText('Superhead');
```

Comment:
because the `waitforelement` is expecting the element to be created. so it only fires when the dom changes. if the element exists before this step runs, it won't pass.

Reply:
Good to know. Thanks for sharing.